drivers/gles3/shaders/scene.glsl

```diff
@@ -726,7 +732,7 @@ void light_compute(vec3 N, vec3 L, vec3 V, float A, vec3 light_color, float atte
 		float D = D_GGX(cNdotH, alpha_ggx);
 		float G = V_GGX(cNdotL, cNdotV, alpha_ggx);
 #endif // LIGHT_ANISOTROPY_USED
-	   // F
+		// F
```

Comment:
clang-format wants this to be reverted to what it was.

It's being buggy and trying to align the two comments, with the assumption that one tab == 4 spaces.
So it becomes:
```
#endif // LIGHT_ANISOTROPY_USED
             // F
```

Adding an empty line to separate them fixes it.

Reply:
Opened a bug report about this upstream:
- https://github.com/llvm/llvm-project/issues/60098

This case is just bad luck that `#endif // stuff` has the comment starting at column 7, and `<tab><tab>// F` has it starting at column 8. This looks like it's just off by one by mistake so clang-format decides to align them (despite us asking it not to with the `AlignTrailingComments: false` option).

Reply:
Thats pretty annoying. I'm pretty sure we're going to have a repeat of this a few more times :)